guidance/models/_model.py

```diff
@@ -865,9 +868,11 @@ def _update_display(self, throttle=True):
                     return # we are throttling the update
                 else:
                     self._last_display = curr_time
-        
-            clear_output(wait=True)
-            display(HTML(self._html()))
+            if ipython_is_imported:
+                clear_output(wait=True)
+                display(HTML(self._html()))
+            else:
+                pprint(self._state)
```

Comment:
pprint is the best I could think of. Should fix https://github.com/guidance-ai/guidance/issues/708